src/autocomplete/EmojiProvider.js

```diff
@@ -24,10 +24,43 @@ import sdk from '../index';
 import {PillCompletion} from './Components';
 import type {SelectionRange, Completion} from './Autocompleter';
 
+import EmojiData from 'emoji-datasource/emoji';
```

Comment:
emoji.json from that package is 874KB - I think this is going to be too big to put in the bundle, realistically